rest.yaml

```
@@ -249,12 +249,926 @@ paths:
+            format: "uuid"
+      responses:
+        '200':
+          description: "successful"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Team"  
+
+  /gyms:
+    get:
+     summary: "get List of Gyms"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)